Description: Add nostartfiles link flag

--- pyslalib-1.0.4.orig/setup.py
+++ pyslalib-1.0.4/setup.py
@@ -9,7 +9,8 @@ ext1 = Extension(name = 'pyslalib.slalib
                  sources = ['slalib.pyf']+\
                            glob.glob("*.f")+\
                            glob.glob("*.F"),
-                 libraries = ['python2.7']
+                 libraries = ['python2.7'],
+                 extra_link_args = ["-nostartfiles"],
                  )
 
 if __name__ == "__main__":
@@ -24,5 +25,5 @@ if __name__ == "__main__":
           packages = ['pyslalib'],
           package_dir = {'pyslalib': 'lib'},
           package_data = {'pyslalib': ['docstring_pickle.pkl']},
-          ext_modules = [ext1]
+          ext_modules = [ext1],
           )
