Description: Don't update the doc

--- pyslalib-1.0.4.orig/setup.py
+++ pyslalib-1.0.4/setup.py
@@ -4,12 +4,6 @@ from numpy.distutils.core import setup,
 import pickle
 import get_docstring
 
-# Generate documentation dictionary and save it in "lib/"
-docstring = get_docstring.get_docstring()
-f = open("lib/docstring_pickle.pkl", "wb")
-pickle.dump(docstring, f)
-f.close()
-
 ext1 = Extension(name = 'pyslalib.slalib',
                  include_dirs = ['.'],
                  sources = ['slalib.pyf']+\
