Description: Link to python


--- pyslalib-1.0.4.orig/setup.py
+++ pyslalib-1.0.4/setup.py
@@ -14,7 +14,9 @@ ext1 = Extension(name = 'pyslalib.slalib
                  include_dirs = ['.'],
                  sources = ['slalib.pyf']+\
                            glob.glob("*.f")+\
-                           glob.glob("*.F"))
+                           glob.glob("*.F"),
+                 libraries = ['python2.7']
+                 )
 
 if __name__ == "__main__":
     setup(name = 'pySLALIB',
